assets/app/views/attachPVC.html

```diff
@@ -0,0 +1,198 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+      <alerts alerts="alerts"></alerts>
+      <div class="row">
+        <div class="create-route-icon col-md-2 gutter-top hidden-sm hidden-xs">
+          <span class="fa fa-hdd-o icon-xl"></span>
+        </div>
+        <div class="col-md-8">
+          <h1>Attach Storage</h1>
+          <div>
+            <span class="help-block">
+              Attach an existing persistent volume claim to the pod template of a 
+              <span ng-if="attach.deploymentConfig">deployment config.</span>
+              <span ng-if="attach.deployment">deployment.</span>
+            </span>
+          </div>
+          <div ng-show="!pvcs || !(attach.deploymentConfig || attach.deployment)">Loading...</div>
+          <div ng-show="pvcs && (attach.deploymentConfig || attach.deployment)">
+            <form name="attachPVCForm">
+              <fieldset ng-disabled="disableInputs">
+                <!-- attaching to a deployment config -->
+                <div class="form-group" ng-if="attach.deploymentConfig">
+                  <label for="deployment-config-name" class="required">Deployment config</label>
+                  <input
+                      id="deployment-config-name"
+                      class="form-control"
+                      type="text"
+                      name="deploymentConfig"
+                      ng-model="attach.deploymentConfig.metadata.name"
+                      placeholder="deployment-config"
+                      select-on-focus
+                      autocorrect="off"
+                      autocapitalize="off"
+                      spellcheck="false"
+                      disabled
+                      aria-describedby="deployment-config-name-help">
+                  <div>
+                    <span id="deployment-config-name-help" class="help-block">The pod templates of this deployment config will specify volumes that attach to the persistent volume claim below.</span>
+                  </div>
+                </div>
+
+                <!-- attaching to a deployment -->
+                <div class="form-group" ng-if="attach.deployment">
+                  <label for="deployment-name" class="required">Deployment</label>
+                  <input
+                      id="deployment-name"
+                      class="form-control"
+                      type="text"
+                      name="deployment"
+                      ng-model="attach.deployment.metadata.name"
+                      placeholder="deployment"
+                      select-on-focus
+                      autocorrect="off"
+                      autocapitalize="off"
+                      spellcheck="false"
+                      disabled
```

Comment:
Always disabled until we have "attach to..." in future. Doesn't look very good as static text, what do you think?

![screenshot from 2015-12-22 20-12-31](https://cloud.githubusercontent.com/assets/158611/11967241/61e84ef8-a8e8-11e5-85a3-d2c819035a57.png)